### PR TITLE
incorporated QE feddback comments

### PR DESCRIPTION
**Pull request by aasingh**

### CODE REVIEW

File: topics/first-realm/user.adoc [hunk 1]
```diff
@@ -22,9 +22,9 @@ the page that will allo you to do this.
 .Set Temporary Password
 image:../../{{book.images}}/credentials.png[]
 
-Enter in a new password and retype it within the `Password Confirmation` field.  Once you do this a red `Reset Password`
+Enter a new password and retype it within the `Password Confirmation` field.  Once you do this a red `Reset Password`
 button should reappear.  Clicking on that will reset the user's password to the new one you specified.  Please note that
-this is a temporary password and the user will be required to change it after they first log in.  You can make it permanent
+this is a temporary password and the users are required to change it after they first log in.  You can make it permanent
```
Comment (stianst) [inline topics/first-realm/user.adoc]: Leave as 'user will be'. The change affects a single user, not multiple users.

File: topics/secure-jboss-app/subsystem.adoc [hunk 2]
```diff
@@ -7,22 +7,22 @@ Open this file and search for the following text:
 
 [source,xml]
 ----
-<subsystem xmlns="urn:jboss:domain:keycloak:1.1"/>
+<subsystem xmlns="urn:jboss:domain:keycloak-server:1.1"/>
```
Comment (stianst) [inline topics/secure-jboss-app/subsystem.adoc]: Invalid change, please leave as it was.

File: topics/secure-jboss-app/subsystem.adoc [hunk 3]
```diff
@@ -7,22 +7,22 @@ Open this file and search for the following text:
...
 ----
 
 Modify this a little bit to prepare it for pasting in your template from the `Installation` tab.
 
 [source,xml]
 ----
-<subsystem xmlns="urn:jboss:domain:keycloak:1.1">
+<subsystem xmlns="urn:jboss:domain:keycloak-server:1.1">
```
Comment (stianst) [inline topics/secure-jboss-app/subsystem.adoc]: Invalid change, please leave as it was.

File: topics/secure-jboss-app/subsystem.adoc [hunk 4]
```diff
@@ -7,22 +7,22 @@ Open this file and search for the following text:
...
 </subsystem>
 ----
 
 Within the `subsystem` element, paste in the template.  It will look something like this:
 
 [source,xml]
 ----
-<subsystem xmlns="urn:jboss:domain:keycloak:1.1">
+<subsystem xmlns="urn:jboss:domain:keycloak-server:1.1">
```
Comment (stianst) [inline topics/secure-jboss-app/subsystem.adoc]: Invalid change, please leave as it was.

File: topics/secure-jboss-app/subsystem.adoc [hunk 5]
```diff
@@ -38,7 +38,7 @@ Change the `WAR MODULE NAME` text to be `vanilla` as follows:
...
```
Comment (stianst) [inline topics/secure-jboss-app/subsystem.adoc]: Invalid change, please leave as it was.

File: topics/first-realm/realm.adoc [hunk 6]
```diff
@@ -8,7 +8,7 @@ Login to the {{book.project.name}} Admin Console using the account you created i
 Admin Console Link::
   http://localhost:8080/auth/admin/
 
-Mouse over the top left corner drop down menu that is titled with `Master`.  If you are logged in the master realm
+Mouseover the top left corner drop down menu that is titled with `Master`.  If you are logged in the master realm
```
Comment (stianst) [inline topics/first-realm/realm.adoc]: Neither 'mouse over' or 'mouseover' makes sense to me. Maybe the text should read 'place the mouse over'

**Comment by aasingh** [comment]
changes reverted as per the comments.

**Pull request merged**